ee/secureenclavesigner/secureenclavesigner_test.go

```diff
@@ -0,0 +1,171 @@
+//go:build darwin
+// +build darwin
+
+package secureenclavesigner
+
+import (
+	"context"
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"os/exec"
+	"os/user"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/kolide/kit/ulid"
+	"github.com/kolide/krypto/pkg/challenge"
+	"github.com/kolide/krypto/pkg/echelper"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	testWrappedEnvVarKey = "SECURE_ENCLAVE_TEST_WRAPPED"
+	macOsAppResourceDir  = "./test_app_resources"
+)
+
+func WithBinaryPath(p string) opt {
+	return func(ses *secureEnclaveSigner) {
+		ses.pathToLauncherBinary = p
+	}
+}
+
+// TestSecureEnclaveTestRunner TODO: description
```

Comment:
`TODO: description` still needs to be done?

Reply:
no clue ... removed